test/ctypes/Makefile

```diff
@@ -0,0 +1,76 @@
+INCLUDES = $(KREMLIN_HOME)/kremlib $(DIRS)
+FSTAR_INCLUDES = $(addprefix --include ,$(INCLUDES))
+OUTPUT_DIR = .extracted
+
+CTYPES = $(shell ocamlfind query ctypes)
+
+FSTAR_NO_FLAGS = $(FSTAR_HOME)/bin/fstar.exe --odir $(OUTPUT_DIR) \
+  --cache_checked_modules $(FSTAR_INCLUDES) --cmi \
+  --already_cached 'FStar LowStar C'
+
+FSTAR = $(FSTAR_NO_FLAGS) $(OTHERFLAGS)
+KRML = $(KREMLIN_HOME)/krml
+
+# F* source files to be extracted
+DIRS = .
+ROOTS = $(wildcard $(addsuffix /*.fsti,$(DIRS)) $(addsuffix /*.fst,$(DIRS)))
+
+# Use F* to build dependency graph for ROOTS
+ifndef MAKE_RESTARTS
+.depend: .FORCE
+	@$(FSTAR_NO_FLAGS) --dep full $(ROOTS) --extract '* -Prims -LowStar -Lib.Buffer -Hacl -FStar +FStar.Endianness +FStar.Kremlin.Endianness' > $@
+
+.PHONY: .FORCE
+.FORCE:
+endif
+
+include .depend
+
+ALL_CHECKED_FILES = $(addsuffix .checked,$(ROOTS))
+
+all: Client.exe
```

Comment:
`all` is not the default target as it does not come first... need to invoke `make all` manually